browser/DaisySubscriptions.js

```diff
@@ -326,7 +313,7 @@ export class DaisySubscriptionsOnToken extends DaisySubscriptions {
    *
    * const account = ...; // we recommend getting `account` from [react-metamask](https://github.com/consensys/react-metamask)
    * const token = daisy.loadToken(plan); // web3 contract instance.
-   * const amount = 100000; // defined by user. We recommend a very big number.
+   * const amount = 100000; // defined by user (remember tokens has "decimals").
```

Comment:
Doesn't this have to be a string? Shouldn't we give it 18 decimals to be realistic?